### DISCUSSION TITLE
play a custom voice everytime bot join/leave the channel?

### DISCUSSION THREAD

**Discussion by fanesz** [jagrosh/MusicBot#898] [Ideas]
is that possible to play like 1-2 sec voice from youtube link everytime bot join or leave the channel? 

i don't understand java but its like
if someone request [requstedsong] -> yt search [requstedsong] -> *bot join the channel* -> play [custom voice 1-2sec from yt link] -> play [requstedsong]

thanks!

**Comment by MichailiK** [comment]
"deferring" songs seem to be kind of tedious to implement. The bot is already pretty quick at finding & playing a song

**Comment by ZenTauren** [comment]
This would be a great addition now that Discord Nitro has this for users, I'd have some funny meme sound every time the bot joins.

**Comment by Luttudo** [comment]
it would be a great feature to differentiate when the bot or a regular user joins a call
